### DISCUSSION TITLE
Portal Crashes On Load

### DISCUSSION THREAD

**Discussion by ecotto7** [vanjarosoftware/Vanjaro.Platform#1300] [Q&A]
I have a Vanjaro instance with 2 portals.  One portal is working the other crashes.  For a while it would open and let you log in and it would only crash when hitting the arrow icon on left.  Now it won't open at all.  Nothing in Admin Logs.  Windows error report is not much help either.  Is there something behind the scenes I can do to troubleshoot this further.
[Report.wer.txt](https://github.com/vanjarosoftware/Vanjaro.Platform/files/6528009/Report.wer.txt)

**Comment by meetmandeep** [comment]
Please provide us with the logs from /Portals/_Default/Logs/LogDateTime

Be sure to delete the existing log and then follow the steps to reproduce the error/crash. Attach the log file here.

**Comment by ecotto7** [comment]
Here is the file you requested along with a video in case that is helpful.

Video: https://www.dropbox.com/s/tzdil7c4or93o0d/zoom_0.mp4?dl=0

Thanks,

Elliot

From: Mandeep Singh ***@***.***>
Reply-To: "vanjarosoftware/Vanjaro.Platform" ***@***.***>
Date: Monday, May 24, 2021 at 1:59 AM
To: "vanjarosoftware/Vanjaro.Platform" ***@***.***>
Cc: ***@***.***" ***@***.***>, Author ***@***.***>
Subject: Re: [vanjarosoftware/Vanjaro.Platform] Portal Crashes On Load (#1300)


Please provide us with the logs from /Portals/_Default/Logs/LogDateTime

Be sure to delete the existing log and then follow the steps to reproduce the error/crash. Attach the log file here.

**Reply by ecotto7** [reply]
Here is the log file you requested: https://www.dropbox.com/s/iwbb8iy330g8dc1/2021.05.24.log.resources.txt?dl=0

**Reply by mksmanish79** [reply]
We are unable to duplicate the issue. We have created a Vanjaro instance with 2 portals http://rc.vanjaro.local (default portal) and http://ecotto.vanjaro.local and both portals are working even when we clicking multiple times on the arrow icon on left.

Maybe the issue due to the error logged in log file "DotNetNuke.Entities.Urls.UrlRewriterUtils - System.ArgumentNullException: Value cannot be null."
